things/database.py

```diff
@@ -261,6 +250,21 @@ def get_tasks(  # pylint: disable=R0914
 
         return self.execute_query(sql_query)
 
+    def get_task_by_uuid(self, uuid, count_only=False):
+        """Get a task by uuid. Raise `ValueError` if not found."""
+        where_predicate = "TASK.uuid = ?"
+        sql_query = make_tasks_sql_query(where_predicate)
+        parameters = (uuid,)
```

Comment:
Why the `,`?

Reply:
We want it to be a `tuple` type. Without the comma, `(uuid)` is turned to `uuid`.

Reply:
Ah, I see, I'd have probably used `parameters = tuple(uuid)` but found that a trailing comma is also often used.

Reply:
You're right, it's not as readable, but glad you pointed it out. It might be one of those (more rare) cases, where Python isn't immediately obvious.

Reply:
Also, `tuple(uuid)` doesn't do what you think it does. What you'd want is `tuple([uuid])`.